.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ltex.ignoreRuleInSentence": [
+    {
+      "rule": "KEY_WORDS",
+      "sentence": "^\\QThe key words \"MUST\", \"MUST NOT\", \"REQUIRED\", \"SHALL\", \"SHALL NOT\", \"SHOULD\", \"SHOULD NOT\", \"RECOMMENDED\", \"MAY\", and \"OPTIONAL\" in this document are to be interpreted as described in RFC 2119.\\E$"
+    }
+  ],
+  "ltex.en-US.disabledRules": [
+    "EN_QUOTES"
+  ]
+}
```

Comment:
Should this vscode setting file not checked in?
And curious what does this setting file do?

Reply:
The `.vscode/settings.json` includes project level configuraiton, like ensuring that particular extensions are configured the same way across multiple machines.

Since these settings are project specific (bypassing some generic rules for Language Tool (LTEX) that don't apply in this case) it seems safe to add?